### PR TITLE
Updated README.md for new CLI

### PR DESCRIPTION
**Pull request by Dherse**
Modified README.md to reflect the changes to the CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,18 +123,27 @@ the bleeding edge version with `nix run github:typst/typst -- --version`.
...
-typst --watch file.typ
+typst watch file.typ
+```
+
+You can also add custom font paths for your project and list all of the discovered fonts:
+```sh
+# Adds the given path in the list of font paths to search for
+typst --font-path path/to/fonts compile file.typ
+
+# List all of the discovered fonts
```
[the remark this answers is not in the document]
Comment (laurmaedje) [reply README.md:145]: (sorry for being so nit-picky :D)

[the remark this answers is not in the document]
Comment (laurmaedje) [reply README.md:145]: Ideally, the short description texts in the CLI would also be worded consistently. Compile**s**, Watche**s**, List**s**, and Print**s**.

[the remark this answers is not in the document]
Comment (Dherse) [reply README.md:145]: It's late, I can't spell anymore

**Review by drupol — COMMENTED** [review]
:+1:

**Comment by laurmaedje** [comment]
I think I'll wait with merging this until the next release, so that the Readme is correct for people that download the latest release.

**Comment by Dherse** [comment]
Alright, makes perfect sense :) If they use the latest build, they will get an error message in the console either way

**Pull request merged**